src/com/esotericsoftware/yamlbeans/WriteClassName.java

```diff
@@ -0,0 +1,5 @@
+package com.esotericsoftware.yamlbeans;
+
+public enum WriteClassName {
+	ALWAYS, NEVER, AUTO
```

Comment:
When would you want to always write the class name? Do we need an enum? Can it just be `YamlConfig.WriteConfig` `setClassNames(boolean)`?